src/assets/thirdPartyServices/mozi/gene_annotation_service/form/index.js

```diff
@@ -338,25 +346,27 @@ const AnnotationForm = props => {
           <FormGroup row>
             <FormControlLabel
               control={<Checkbox color="primary" onChange={e => toggleAnnotation("gene-pathway-annotation", e)} />}
-              label="Gene Pathway"
+              label="Curated Pathways"
             />
           </FormGroup>
           {annotations.includes("gene-pathway-annotation") && (
             <div className="annotation-parameters">
               <div className="parameter">
-                {Pathways.map(p => (
-                  <FormControlLabel
-                    control={
-                      <Checkbox
-                        color="primary"
-                        value={p.value}
-                        onChange={e => togglePathways(p.value, e)}
-                        defaultChecked={pathways.includes(p.value)}
-                      />
-                    }
-                    label={p.label}
-                  />
-                ))}
+                <FormControlLabel
+                  control={
+                    <Checkbox
+                      color="primary"
+                      value="reactome"
+                      onChange={e => togglePathways("reactome", e)}
+                      defaultChecked={pathways.includes("reactome")}
+                    />
+                  }
+                  label={
+                    <a href="http://www.reactome.org" target="_blank">
```

Comment:
use the `AnchorLink` component from our common directory `src/components/common/AnchorLink.js`. 
you just have to pass `newTab` as a prop, if you want to open it in new tab, it will take care of the rest.